### PR TITLE
edit document, api and use api user profile

### PR DESCRIPTION
**Pull request by ghost**
## Description
edit document, api and use api user profile
## Migrations
NO

## Impacted Areas in Application
List general components of the application that this PR will affect:

-

## SQL query and `explain`
List of all queries and attach the result of each explaining query (in text, screenshot...)

### CODE REVIEW

**Comment by at-sonle** [comment]
LGTM

File: public/js/user/showProfileUser.js [hunk 1]
```diff
@@ -8,11 +8,19 @@ $( document ).ready(function() {
...
     headers: { 'authorization': 'Bearer '+ localStorage.getItem('access_token') },
     success: function (response){
+      var shippings = response.data.user.shippings;
+      var html = '';
       $("#user-avatar").attr('src', response.data.user.user_info.avatar_url);
       $("#user-name").html(response.data.user.name);
       $("#user-address").html(response.data.user.user_info.address);
       $("#user-phone").html(response.data.user.user_info.phone);
       $("#user-email").html(response.data.user.email);
+      if (shippings) {
```
Comment (at-tuanphan) [inline public/js/user/showProfileUser.js]: anh nghĩ trong `foreach` của JS nó có check empty rồi thì phải, nên chắc ko cần `if` đâu em

**Review by ghost — COMMENTED** [review]
please check my comment

**Review by ghost — APPROVED** [review]
LGTM

**Pull request merged**